src/views/main.js

```diff
@@ -47,7 +47,7 @@ const mainLayout = data => `
     <!-- End Google Tag Manager -->
     </head>
   <body>
-    ${data.partial.name === 'landing' ? landingHeader(data) : mainHeader(data)}
+    ${['landing', 'notFound'].includes(data.partial.name) ? landingHeader(data) : mainHeader(data)}
```

Comment:
With the addition of this error page we should probably create a separate layout to handle these two partials and the others to come that don't require UI for logged-in user.  I created the following task to track this:
https://mozilla-hub.atlassian.net/browse/MNTOR-1254